homework/index.js

```diff
@@ -0,0 +1,56 @@
+const status = (response) => {
+  if (response.status >= 200 && response.status < 300) {
+    return Promise.resolve(response);
+  }
+  return Promise.reject(new Error(response.statusText));
+};
+const json = response => response.json();
+window.onload = () => fetch('https://api.github.com/orgs/HackYourFuture/repos?per_page=100')
+  .then(status)
+  .then(json)
+  .then((data) => {
```

Comment:
Better use `repos` or something as name; so it's more clear what this is further down the code.